Gems/O3DE/GeomNodes/Code/Source/Editor/Rendering/GNMeshController.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+#include <AzFramework/Visibility/BoundsBus.h>
+#include <AzFramework/Asset/AssetCatalogBus.h>
+#include <AzCore/Component/TransformBus.h>
+#include <AzToolsFramework/API/ComponentEntitySelectionBus.h>
+#include <Editor/Rendering/GNModelData.h>
+
+namespace GeomNodes
+{
+	class GNRenderMesh;
+	//! A common class that handles everything from storing the model data, rendering the mesh via GNRenderMesh
+	//! handling different mesh events like bound request, transform updates and others.
```

Comment:
nit: indentation is inconsistent

Reply:
noted. Might be resolved when `clang-format` is used as Alex suggested

Reply:
fixed in commit [d756b38](https://github.com/o3de/o3de-technicalart/pull/11/commits/d756b38541172fd3e0051140ca96cc9bd8b05388)